lightning/src/chain/channelmonitor.rs

```diff
@@ -104,16 +102,6 @@ pub struct ChannelMonitorUpdate {
 	pub channel_id: Option<ChannelId>,
 }
 
-/// The update ID used for a [`ChannelMonitorUpdate`] that is either:
-///
-///	(1) attempting to force close the channel by broadcasting our latest commitment transaction or
-///	(2) providing a preimage (after the channel has been force closed) from a forward link that
-///		allows us to spend an HTLC output on this channel's (the backward link's) broadcasted
-///		commitment transaction.
-///
-/// No other [`ChannelMonitorUpdate`]s are allowed after force-close.
-pub const CLOSED_CHANNEL_UPDATE_ID: u64 = core::u64::MAX;
```

Comment:
> I'm a bit hesitant to leave a random constant dangling forever

It seems like we're doing that anyway (internally at least), except it's a magic number now instead of a documented constant? Seems like we could just use it internally and not expose it in the public API. Unless I'm missing that we can migrate away from using `u64::MAX` everywhere in the short-medium term?